### PR TITLE
Address peak sensible load with sizing factor at cooling peak conditions reports zero value

### LINKED ISSUE #9962: AirLoop Component Load Summary - Peak Sensible Load with Sizing Factor at Cooling Peak Conditions reports 0
Issue overview
--------------
Running a simulation with the standard Coil:Cooling:Water on an Air Loop. Using the AirLoop component Load Summary inspect the Cooling Peak Condition Table. Regardless of the Sizing Factor used in the model the value reported here is 0. Attached is an example project with the cooling sizing factor set to 1.15. On this project the expected value would be 3654.7 watts. 

### Details
Some additional details for this issue (if relevant):
 - Platform (Windows 10)
 - Version of EnergyPlus 22.2.0
 

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [run.zip](https://github.com/NREL/EnergyPlus/files/11225863/run.zip) - example file showing behavior
 - 
image html report
image sql values.

### PR DESCRIPTION
**Pull request by Nigusse**
Pull request overview
---------------------
 - Fixes #9962 
 - Fixes `Peak Sensible Load with Sizing Factor` value reported to `AirLoop component Load Summary` report, `Cooling Peak Condition` table. Diffs are expected only if the `sizing:system` object `Type of Load to Size On` field is set to `Total`.

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [x] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [x] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [x] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [x] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/EnergyPlus/SimAirServingZones.cc [hunk 1]
```diff
@@ -6370,6 +6370,8 @@ void UpdateSysSizing(EnergyPlusData &state, Constant::CallIndicator const CallIn
                         state.dataSize->CalcSysSizing(AirLoopNum).SysCoolOutHumRatSeq = sysSizing.SysCoolOutHumRatSeq;
                         state.dataSize->CalcSysSizing(AirLoopNum).SysDOASHeatAddSeq = sysSizing.SysDOASHeatAddSeq;
                         state.dataSize->CalcSysSizing(AirLoopNum).SysDOASLatAddSeq = sysSizing.SysDOASLatAddSeq;
+                        state.dataSize->CalcSysSizing(AirLoopNum).SysDesCoolLoad = sysSizing.SysDesCoolLoad;
+                        state.dataSize->CalcSysSizing(AirLoopNum).SysCoolLoadTimeStepPk = sysSizing.SysCoolLoadTimeStepPk;
```
Comment (Nigusse) [inline src/EnergyPlus/SimAirServingZones.cc:6374]: System sizing variables `SysDesCoolLoad` and `SysCoolLoadTimeStepPk` were not set values when the system `Type of Load to Size On` is `Total`.

File: tst/EnergyPlus/unit/OutputReportTabular.unit.cc [hunk 2]
```diff
@@ -10904,3 +10904,936 @@ TEST_F(SQLiteFixture, WriteVeriSumSpaceTables_Test)
...
+    // get the 'Peak Sensible Load with Sizing Factor'
+    std::string query("SELECT Value From TabularDataWithStrings"
+                      "  WHERE TableName = 'Cooling Peak Conditions'"
+                      "  AND ReportName = 'AirLoop Component Load Summary'"
+                      "  AND ColumnName = 'Value'"
+                      "  AND RowName = 'Peak Sensible Load with Sizing Factor'");
+    // check the value from result records
+    Real64 return_val = execAndReturnFirstDouble(query);
+    EXPECT_EQ(return_val, 5080.22);
+}
```
Comment (Nigusse) [inline tst/EnergyPlus/unit/OutputReportTabular.unit.cc:12689]: This unit test illustrates that the `Peak Sensible Load with Sizing Factor` value set correctly when the system sizing load type is `Total.`

**Review by Nigusse — COMMENTED** [review]
Code change walk though:
- System sizing now correctly sets the peak load value for reporting to `AirLoop Component Load Summary` report name and `Cooling Peak Conditions` table.

**Comment by Nigusse** [comment]
Sample result after the fix based on the defect file provided. Now the `Peak Sensible Load with Sizing Factor` value is `3654.59W` and the `Difference Due to Sizing Factor` is also corrected to `476.69W` = `(3654.59 -  3177.90)W`.

image

**Comment by Nigusse** [comment]
The SQLITE output file also shows the same result:

image

**Comment by nrel-bot-2c** [comment]
@Nigusse it has been 9 days since this pull request was last updated.

**Comment by nrel-bot-2** [comment]
@Nigusse it has been 7 days since this pull request was last updated.

**Review by rraustad — APPROVED** [review]
This looks like the correct code change.

**Comment by nrel-bot-2b** [comment]
@Nigusse it has been 7 days since this pull request was last updated.

**Comment by nrel-bot-2c** [comment]
@Nigusse it has been 7 days since this pull request was last updated.

**Comment by Nigusse** [comment]
The regression test suite files do not show diffs because the `Type of Load to Size On` field in `Sizing:System` object in  all example files were set to `Sensible`. The unit test was created with the `Type of Load to Size On` set to `Total`.

**Comment by nrel-bot-2** [comment]
@Nigusse it has been 9 days since this pull request was last updated.

**Comment by Myoldmopar** [comment]
This looks reasonable to me as well.  I'm glad things are matching up.  There is a conflict, so I will try to pull develop and resolve it.  OK I did that, the conflict was that Git was confused about two different but similar unit tests added to the ORT.unit.cc file.  I made sure both were in there and everything seems good.  Happy to merge this shortly.

**Comment by Myoldmopar** [comment]
Enough checks have run, merging this, thanks @Nigusse

**Pull request merged**